ingestion/src/metadata/ingestion/sink/metadata_rest.py

```diff
@@ -170,7 +170,8 @@ def write_datamodel(self, datamodel_link: DataModelLink) -> None:
             )
         else:
             logger.warning(
-                f"Could not find any entity by Table FQN [{datamodel_link.fqn}] when adding DBT models."
+                f"The table [{datamodel_link.fqn.__root__}] from the manifest file is not found in OM. "
+                f"Please check if the table is ingestion"
```

Comment:
```suggestion
                "Please, check if the table has been ingested previously."
```

Reply:
Made the changes thanks @nahuelverdugo 👍